src/openzaak/utils/validators.py

```diff
@@ -268,3 +272,18 @@ def __call__(self, url: str):
             raise error
 
         return obj
+
+
+class UniekeIdentificatieValidator(_UniekeIdentificatieValidator):
+
+    partial_message = _(
+        "Deze identificatie ({identificatie}) bestaat al binnen de organisatie"
+    )
+
+    @property
+    def message(self):
+        return self.partial_message.format(identificatie=self.identificatie)
+
+    def __call__(self, attrs: dict, serializer):
+        self.identificatie = attrs.get(self.identificatie_field)
+        return super().__call__(attrs, serializer)
```

Comment:
IIRC this is subject to a race condition because the same validator instance can and will be used for different requests (see https://www.django-rest-framework.org/community/3.11-announcement/#validator-default-context)